cfgov/jinja2/v1/test-fixture/index.html

```diff
@@ -0,0 +1,116 @@
+
+    <!-- Open Graph properties -->
+        <!--   Required  -->
+        <meta property="og:title" content="{% block og_title %}{{ self.title() }}{% endblock %}">
+        <meta property="og:type" content="{% block og_type %}website{% endblock %}">
+        <meta property="og:url" content="{{ request.url }}">
+        <meta property="og:image"
+              content="http://www.consumerfinance.gov/wp-content/themes/cfpb_nemo/_/img/logo.svg">
+        <!--   Optional -->
+        <meta property="og:description" content="{% block og_desc %}{{ self.desc() }}{% endblock %}">
+        <meta property="og:site_name" content="Consumer Financial Protection Bureau">
+        <!--   Facebook -->
+        <meta property="fb:app_id" content="210516218981921">
+        {% block og_article_author %}{% endblock %}
+    <!-- End of Open Graph properties -->
+
+    {# TODO: Explicit favicon link needed for testing. Remove for production. #}
+    <link rel="shortcut icon" type="image/x-icon" href="/favicon.ico">
+
+<!--
+    ======
+    STYLES
+    ======
+    The number of stylesheets here must be kept to a minimum.
+    Unless adding a significant amount of CSS that is specific to a single page or section of the site,
+    all new styles should be added to cfgov/v1/unprocessed/css/main.less.
+-->
+
+{% block css %}
+<!--[if lt IE 9]>      <link rel="stylesheet" href="{{ static('css/main.ie.css') }}"> <![endif]-->
+<!--[if gt IE 8]><!--> <link rel="stylesheet" href="{{ static('css/' + atomic_type + '.css') }}"> <!--<![endif]-->
+{% endblock css %}
+
+<!--
+    ============
+    HEAD SCRIPTS
+    ============
+    The following scripts must be included in the head. DO NOT ADD ANY MORE.
+    All other scripts should go before the closing body tag.
+    If you come across a script that makes a convincing case to be included in
+    the head, then file an issue or PR to discuss including it.
+-->
+    {# Customized Modernizr build that includes html5shiv.
+       Built via gulp-modernizer in `scripts.js` task. #}
+    <script src="{{ static('js/modernizr.min.js') }}"></script>
+
+    <!--[if lt IE 9]>
+    <script>
+        // If in IE8 reverse no-js/js class change made by modernizr.
+        var docElement = document.documentElement;
+        docElement.className = docElement.className.replace( /(^|\s)js(\s|$)/, '$1no-js$2' );
+    </script>
+    <![endif]-->
+
+</head>
+
+<body>
+
+{% if atomic_type == 'header' %}
+
+    {# Overlay for the page. Used for the mobile mega menu. #}
+    <div class="a-overlay u-hidden"></div>
+
+    {% import 'organisms/header.html' as o_header with context %}
+    {{ o_header.render( show_banner=false ) }}
+
+{% endif %}
+
+<!--
+    ============
+    BODY SCRIPTS
+    ============
+    The number of scripts here must be kept to a minimum.
+-->
+
+{% block javascript %}
+<script type='text/javascript'>
+    {% include '/js/atomic/' + atomic_type + '.js' %}
```

Comment:
Added in https://github.com/cfpb/cfgov-refresh/pull/1796/commits/245870449d5108bb31cc96376fa5d2be7cf74927